data/139/359/563/9/1393595639.geojson

```diff
@@ -72,8 +81,8 @@
         }
     ],
     "wof:id":1393595639,
-    "wof:lastmodified":1566715004,
-    "wof:name":"H\u00fcllenerfehn",
+    "wof:lastmodified":1703098789,
+    "wof:name":"V\u00f6llenerfehn",
```

Comment:
Nit: Enlgish wof:name is preferred, so `Vollenerfehn` would be better here? But it was already not English so nice to have not blocker.

Reply:
Yeah, that's a good point.. I'm going to close this PR and re-open a new one with this change included.